docs/theming/developers.md

```diff
@@ -35,24 +35,24 @@ hasToc: true
 
 ## Using the theming system
 
-<abbr title="Red Hat Design System">RHDS</abbr>' theming system is a high-level 
+<abbr title="Red Hat Design System">RHDS</abbr>’ theming system is a high-level 
```

Comment:
`RHDS’` → `RHDS’s `

If I'm reading this right, I think I'd add a colon to this sentence, or at the very least remove the comma after "tokens":

`components of the system: [tokens](https://deploy-preview-1937--red-hat-design-system.netlify.app/tokens/) and [elements](https://deploy-preview-1937--red-hat-design-system.netlify.app/elements/).`

Reply:
https://github.com/RedHat-UX/red-hat-design-system/pull/1937/commits/f29dd9c1ef1cc639b97eefc0eecc8d30f68b0d43